NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export(rolldown_style_default)
```

Comment:
Could you do only one thing per PR and not commit this change? https://yihui.name/en/2018/02/bite-sized-pull-requests/